deepmd/pt/utils/env.py

```diff
@@ -34,6 +39,47 @@
 CACHE_PER_SYS = 5  # keep at most so many sets per sys in memory
 ENERGY_BIAS_TRAINABLE = True
 
+ACTIVATION_FN_DICT = {
+    "relu": F.relu,
+    "relu6": F.relu6,
+    "softplus": F.softplus,
+    "sigmoid": torch.sigmoid,
+    "tanh": torch.tanh,
+    "gelu": F.gelu,
+    # PyTorch has no gelu_tf
+    "gelu_tf": lambda x: x * 0.5 * (1.0 + torch.erf(x / 1.41421)),
+    "linear": lambda x: x,
+    "none": lambda x: x,
+}
```

Comment:
You can use this class: 

https://github.com/deepmodeling/deepmd-kit/blob/0fa1b435355cbcfe10af8e766a38fe34c502738f/deepmd/pt/utils/utils.py#L21-L46